coverage.py

```diff
@@ -803,8 +803,12 @@ def items_that_need_coverage(self, identifiers=None, **kwargs):
             identifiers=self.input_identifiers, collection=self.collection_or_not,
             **kwargs
         )
+
         if identifiers:
             qu = qu.filter(Identifier.id.in_([x.id for x in identifiers]))
+        if not identifiers and identifiers != None:
```

Comment:
Caught this when I was running tests. I'm assuming here that if an empty list is given, the provider shouldn't be run on all missing identifiers.